lib/bbb/credentials.rb

```diff
@@ -113,7 +113,8 @@ def parse_response(response)
     def encoded_url(endpoint, secret, params)
       encoded_params = params.to_param
       string = "getUser#{encoded_params}#{secret}"
-      checksum = OpenSSL::Digest.digest('sha1', string).unpack1('H*')
+      checksum_algorithm = Rails.configuration.checksum_algorithm
+      checksum = OpenSSL::Digest.digest(checksum_algorithm, string).unpack1('H*')
```

Comment:
lowercase the `checksum_algorithm` in case the sysadmin used CAPITAL or Capitalized strings

Reply:
Done, except I did it when setting the config variable in application.rb